manifests/params.pp

```diff
@@ -156,16 +156,20 @@
 
   # Service Settings for SystemD in EL7
   if $::osfamily == 'RedHat' and $::operatingsystemmajrelease == '7' {
-    $webhook_service_file     = '/usr/lib/systemd/system/webhook.service'
+    $webhook_service_file     = undef
```

Comment:
instead of defining this for each OS, can we just use the service_provider fact from stdlib?

Reply:
The idea is let 'systemd' module handle the part you selected

As for the default service provider, sure, I can do that